packages/@ngtools/webpack/README.md

```diff
@@ -29,7 +30,30 @@ exports = { /* ... */
 
 The loader works with the webpack plugin to compile your TypeScript. It's important to include both, and to not include any other TypeScript compiler loader.
 
-For Angular version 5 and up, import `AngularCompilerPlugin` instead of `AotPlugin`.
+For Angular version 5 and up, use `AngularCompilerPlugin` instead of `AotPlugin`:
```

Comment:
You should say that AotPlugin will not be supported at all. Period. It's not just a suggestion.